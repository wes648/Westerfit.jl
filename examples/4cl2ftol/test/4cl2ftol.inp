4cl2ftol   @   2024-03-07T16:27:23.596
%CNTRLS
NFOLD = 3
S = 1.5
TK = 25.0
RUNmode = F
mcalc = 5
vtmax = 0
assign = RAM36
turducken = 1
Jmax = 16.5
νmin = 4.0
νmax = 23.0

%2NDORDER
A;             3028.963881355823;   1.0
B;             866.5049228604415;   1.0
C;             672.4557318078677;   1.0
Dab;         -126.68257903237395;   1.0
ϵzz;                         0.0;   0.0
ϵxx;                         0.0;   0.0
ϵyy;                        -0.0;   0.0
ϵxz;                        -0.0;   0.0
χzz;          -69.21833810747393;   1.0
χxz;                         0.0;   0.0
χxmy;         2.4180565623255466;   1.0
F;            5.3925209999999995;   0.0
ρ;          0.018425246526301408;   1.0
Vn;           224.17612807809692;   1.0
η;                           0.0;   0.0

%PARAMS N^a Nz^b (N₊^c + N₋^c) (NS)^d Sz^e Pₐ^f cos(g*α) sin(h*α) Ny^(1-δ(0,h))
%Op;                         Val;   Scl;   a;   b;   c;   d;   e;   f;   g;   h;  stg
μa;                          1.0;   0.0;   0;   1;   0;   0;   0;   0;   0;   0;   0
μa(3);                     0.008;   0.0;   0;   1;   0;   0;   0;   0;   3;   0;   0
μb;                          1.0;   0.0;   0;   0;   1;   0;   0;   0;   0;   0;   0
μb(3);                     0.008;   0.0;   0;   0;   1;   0;   0;   0;   3;   0;   0
μc(3);                    -0.034;   0.0;   0;   0;   0;   0;   0;   0;   0;   3;   0
DN;        -8.097181188832632e-6;   1.0;   4;   0;   0;   0;   0;   0;   0;   0;   1
V3N;          -0.379869340581063;   1.0;   2;   0;   0;   0;   0;   0;   0;   0;   1
V3N;                        -1.0;   1.0;   2;   0;   0;   0;   0;   0;   3;   0;  -1
