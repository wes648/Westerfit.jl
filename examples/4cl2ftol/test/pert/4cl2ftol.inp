4cl2ftol   @   2024-03-10T15:00:25.109
%CNTRLS
NFOLD = 3
S = 1.5
TK = 25.0
RUNmode = F
mcalc = 5
vtmax = 0
assign = RAM36
turducken = 3
Jmax = 16.5
νmin = 4.0
νmax = 23.0

%2NDORDER
A;             3029.049791666588;   1.0
B;             866.4144775949173;   1.0
C;             672.4546719484738;   1.0
Dab;          -125.9166803911434;   1.0
ϵzz;                         0.0;   0.0
ϵxx;                         0.0;   0.0
ϵyy;                        -0.0;   0.0
ϵxz;                        -0.0;   0.0
χzz;           -69.4442107025584;   1.0
χxz;                         0.0;   0.0
χxmy;         2.3182982335533255;   1.0
F;            5.3925209999999995;   0.0
ρ;          0.018432092134039547;   1.0
Vn;           224.20864279830928;   1.0
η;                           0.0;   0.0

%PARAMS N^a Nz^b (N₊^c + N₋^c) (NS)^d Sz^e Pₐ^f cos(g*α) sin(h*α) Ny^(1-δ(0,h))
%Op;                         Val;   Scl;   a;   b;   c;   d;   e;   f;   g;   h;  stg
μa;                          1.0;   0.0;   0;   1;   0;   0;   0;   0;   0;   0;   0
μa(3);                     0.008;   0.0;   0;   1;   0;   0;   0;   0;   3;   0;   0
μb;                          1.0;   0.0;   0;   0;   1;   0;   0;   0;   0;   0;   0
μb(3);                     0.008;   0.0;   0;   0;   1;   0;   0;   0;   3;   0;   0
μc(3);                    -0.034;   0.0;   0;   0;   0;   0;   0;   0;   0;   3;   0
DN;        -8.068667505235911e-6;   1.0;   4;   0;   0;   0;   0;   0;   0;   0;   1
V3N;        -0.37543118255513536;   1.0;   2;   0;   0;   0;   0;   0;   0;   0;   1
V3N;                        -1.0;   1.0;   2;   0;   0;   0;   0;   0;   3;   0;  -1
