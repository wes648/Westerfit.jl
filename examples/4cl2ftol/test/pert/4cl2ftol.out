4cl2ftol   @   2024-03-10T15:00:08.499

Control Parameters
mcalc = 5
νmax = 23.0
maxiter = 60
overwrite = true
Irrep = Ir
λlm0 = 0.0001
apology = true
Jmax = 16.5
S = 1.5
assign = RAM36
turducken = 3
REJECT = 10.0
TK = 25.0
RUNmode =  F
vtmax = 0
NFOLD = 3
νmin = 4.0
INTthres = 1.0e-5

Initial Parameters
 BK;             2314.5394766439;   1.0
 BN;           769.4345561850123;   1.0
 B⨦;           48.48994090925561;   1.0
 Dab;        -125.91632027774016;   1.0
 T⁰₀(ϵ);                    -0.0;   0.0
 T²₀(ϵ);                     0.0;   0.0
 T²₁(ϵ);                     0.0;   0.0
 T²₂(ϵ);                     0.0;   0.0
 T²₀(χ);      -69.44420903396295;   1.0
 T²₁(χ);                    -0.0;   0.0
 T²₂(χ);      0.9464405446276783;   1.0
 F;            161663.7125406618;   0.0
 -2ρF;        -5959.613995392647;   1.0
 V3/2;      3.3608053962369747e6;   1.0
 η;                          0.0;   0.0

 DN;        -8.068690666011732e-6;   1.0
 V3N;         -0.3754416376445879;   1.0
 V3N;                        -1.0;   1.0

Number of lines = 312

Initial RMS = 0.0073814232130360756 MHz
Initial λ = 7.381395970532839e-10

-------------------------------------

             2314.53917322492;    -0.0003
 BN;            769.4345747716956;     0.0000
 B⨦;             48.4899514116109;     0.0000
 Dab;          -125.9166803911434;    -0.0004
 T⁰₀(ϵ);                     -0.0;      Fixed
 T²₀(ϵ);                      0.0;      Fixed
 T²₁(ϵ);                      0.0;      Fixed
 T²₂(ϵ);                      0.0;      Fixed
 T²₀(χ);        -69.4442107025584;    -0.0000
 T²₁(χ);                     -0.0;      Fixed
 T²₂(χ);       0.9464412906335387;     0.0000
 F;             161663.7125406618;      Fixed
 -2ρF;         -5959.600888560725;     0.0131
 V3/2;        3.360803006467457e6;    -2.3898
 η;                           0.0;      Fixed
 DN;        -8.068667505235911e-6;     0.0000
 V3N;        -0.37543118255513536;     0.0000
 V3N;                        -1.0;      Fixed

Max Change; V3N, 0.0028%

-------------------------------------

 The RMS has stopped decreasing. Hopefully it is low.

After   1 iterations:

Final RMS = 0.0074 MHz
Final log₁₀(λ) = -9.1319


                         Parameter                   Uncertainty
 A;             3029.049791666588;           0.24812930714479126
 B;             866.4144775949173;           0.11262411014007331
 C;             672.4546719484738;           0.11262411014007331
 Dab;          -125.9166803911434;            1.3438162131520457
 ϵzz;                         0.0;                           0.0
 ϵxx;                         0.0;                           0.0
 ϵxz;                        -0.0;                           0.0
 ϵyy;                        -0.0;                           0.0
 χzz;           -69.4442107025584;           0.07154540306166754
 χxz;                         0.0;                           0.0
 χxx-χyy;      2.3182982335533255;           0.04349434176917061
 F;            5.3925209999999995;                           0.0
 ρ;          0.018432092134039547;          3.058411006732704e-5
 V3;           224.20864279830928;           0.11491307846193236
 η;                           0.0;                           0.0
 DN;        -8.068667505235911e-6;         2.1396830878521863e-7
 V3N;        -0.37543118255513536;          0.018411035872175487
 V3N;                        -1.0;                           0.0


Journal Formatted Values
 A;                 3.02905(25)e3
 B;                  866.41(11)e0
 C;                  672.45(11)e0
 Dab;                -125.9(13)e0
 ϵzz;               0.0e0 (Fixed)
 ϵxx;               0.0e0 (Fixed)
 ϵxz;              -0.0e0 (Fixed)
 ϵyy;              -0.0e0 (Fixed)
 χzz;               -69.444(72)e0
 χxz;               0.0e0 (Fixed)
 χxx-χyy;             2.318(43)e0
 F;             5.39252e0 (Fixed)
 ρ;                 18.432(31)e-3
 V3;                 224.21(11)e0
 η;                 0.0e0 (Fixed)
 DN;                 -8.07(21)e-6
 V3N;               -375.0(18)e-3
 V3N;               -1.e0 (Fixed)


 Again, sorry about the name...


